docs/production-deployment/cloud/account-setup/namespaces.mdx

```diff
@@ -317,9 +317,10 @@ Each Namespace in Temporal Cloud has two unique endpoints, both of which include
 To ensure the security of your data, all traffic to and from your Namespace is encrypted.
 However, for enhanced protection, you have additional options:
 
-- (recommended) Set up [PrivateLink](https://aws.amazon.com/privatelink/) by [creating a ticket for Temporal Support](/cloud/support#support-ticket).
-  If you don't host your Clients and Workers on AWS, you can create an AWS account to proxy traffic through.
-- Allow list, for outgoing network requests from your Clients and Workers, the [IP address ranges](https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html) of the AWS region in which your Namespace is located.
+- (recommended) Set up private connectivity by [creating a ticket for Temporal Support](/cloud/support#support-ticket).
+- Allow list, for outgoing network requests from your Clients and Workers, the [IP address ranges](https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html) of the cloud provider region in which your Namespace is located.
+  - [AWS IP address ranges](https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html)
```

Comment:
You made me look.

When talking about cloud providers is the term normally uppercased in technical  communication?

In technical communication, terms like "Cloud Provider" or "Cloud Service Provider" are often uppercased to denote specific entities or categories, especially when referring to major players like Amazon Web Services (AWS), Microsoft Azure, or Google Cloud Platform (GCP). Using uppercase helps differentiate these terms from general usage. For instance:

Cloud Provider (when referring to a specific provider or category)
Cloud Service Provider (for a formal category)
However, in more general contexts, lowercase might be used if the term is not meant to specify a particular entity or if it’s used generically. For example:

"Our application can be deployed on any cloud provider."
Ultimately, the choice may depend on the style guide or conventions used by the organization or document.

Reply:
Upper cased.